adding cfp (#233)
USRSE/usrse.github.io: _events/2020/2020-07-pearc20.md

Join us at PEARC20 for a workshop focused on Research Software Engineers.  

> The **Research Software Engineers Community Workshop** will be July 31st, starting a 8am PDT (Portland, OR time).  It is open to all [PEARC20](https://pearc.acm.org/pearc20/) registrants.  

Reviewer: ```suggestion
The **Research Software Engineers Community Workshop** will be July 31st, starting at 8 am PDT (Portland, OR time).  It is open to all [PEARC20](https://pearc.acm.org/pearc20/) registrants.  
```

The full PEARC20 conference runs July 26-31 and is virtual this year.  Information about the conference is available on the [PEARC20

> conference website](https://pearc.acm.org/pearc20/).  [PEARC20 Registration](http://www.cvent.com/events/practice-and-experience-in-advanced-research-computing-2020-pearc-/event-summary-e4cda8bfe4c545b7bb3e458e13a128c2.aspx?tw=12-7B-8D-67-26-C9-CC-DD-AD-82-E2-F8-E0-D3-05-97) will be required for all workshop participants.

Reviewer 2: Do you know what the `?tw=12-7B-8D-67-26-C9-CC-DD-AD-82-E2-F8-E0-D3-05-97` get param is? It seems to work without it, so in case this is a link specific to you (or some other metadata that shouldn't be shared) it might work to just remove it.

The workshop will provide an opportunity for
research software engineers to connect and share their work.  There will be
presentations from RSEs about their software, research, projects, and groups, and about
the development of the RSE community.  There will also be discussion of challenges facing RSEs and what the community can do to help meet those challenges.

> You can participate in the workshop by giving a short talk or joining as a general participant.  To give a talk, please see the guidelines below.  To join as a participant, [register for PEARC20](http://www.cvent.com/events/practice-and-experience-in-advanced-research-computing-2020-pearc-/event-summary-e4cda8bfe4c545b7bb3e458e13a128c2.aspx) and then join our workshop session on Friday.

Reviewer: ```suggestion
You can participate in the workshop by giving a short talk or joining as a general participant.  To give a talk, please see the guidelines below.  To join as a participant, [register for PEARC20](http://www.cvent.com/events/practice-and-experience-in-advanced-research-computing-2020-pearc-/event-summary-e4cda8bfe4c545b7bb3e458e13a128c2.aspx) and then join our workshop session on Friday, July 31st.
```

> You can participate in the workshop by giving a short talk or joining as a general participant.  To give a talk, please see the guidelines below.  To join as a participant, [register for PEARC20](http://www.cvent.com/events/practice-and-experience-in-advanced-research-computing-2020-pearc-/event-summary-e4cda8bfe4c545b7bb3e458e13a128c2.aspx?tw=12-7B-8D-67-26-C9-CC-DD-AD-82-E2-F8-E0-D3-05-97) and then join our workshop session on Friday.

Reviewer 2: Same here.

### Propose a Talk

Are you a research software engineer?  Are you a researcher or student writing software?  Are you a software developer writing software for research?  We want to hear and learn from you at the workshop.

Possible talk topics:

* How you solved a problem on a project
* The research impact of software you helped create
* How software you helped develop could help other RSEs or researchers
* A framework or package that has helped you solve a problem
* A challenge or opportunity for research software in a particular research field
* How research software engineers are organized at your institution
* Teaching software engineering skills to others
* Your RSE career path
* Opportunities and challenges for research software and RSEs overall (sustainability, culture, citation)

Talks from all research fields and concerning all types of technology are welcome. Group submissions with multiple authors are encouraged, however we ask that you designate only one author as presenter.  

> Proposals from individuals who may be first time conference presenters or consider themselves to be part of an underrepresented group in the research software engineering community are particularly encouraged.  

Reviewer: ```suggestion
Proposals from individuals who may be first-time conference presenters or consider themselves to be part of a group that is underrepresented in the research software engineering community are particularly encouraged.  
```

Reviewer: Also see https://cacm.acm.org/blogs/blog-cacm/245710-underrepresented-minority-considered-harmful-racist-language/fulltext for some thoughts on language

Reviewer 3: @danielskatz Do you have suggestions for alternate language here?  I'm writing a similar section in the SC20 call and am not sure what to write.

Reviewer 3: Or is your suggested change here considered sufficient?

Reviewer: I think this is the best simple change. I don't have a good solution overall.

> 

Reviewer 2: It's probably implied that talk == join into a virtual meeting / Zoom and then share slides, but maybe we should state that explicitly (e.g., you could imagine recording in advance, or creating a presentation different from standard slides).

Talks can be 5, 10, or 20 minutes in length and will be presented live during the workshop through the conference platform Brella.  

> Submitted talks will be reviewed on the basis of impact on the community and how they contribute to program with a diverse group of speakers and topics.

Reviewer: ```suggestion
Submitted talks will be reviewed on the basis of impact on the community and how they contribute to a program with a diverse group of speakers and topics.
```

Speakers do not need to be members of the US-RSE Association, although we welcome you to [join](https://us-rse.org/join/).

### Submission Guidelines

> To present a talk at the workshop, [submit](https://easychair.org/conferences/?conf=pearc20rse) a title and "abstract" via EasyChair.

Reviewer 2: Noooooooo not EasyChair! _dramatically falls over_ :laughing:

Please include in the abstract field of the submission form:

* A short description of your talk.

> * If applicable, the related research fields (e.g. humanities, engineering, HPC, computer science, biology, etc.).

Reviewer: ```suggestion
* If applicable, the related research fields (e.g., humanities, engineering, HPC, computer science, biology, etc.)
```

* If you are a student, please indicate so.  Student talks are welcome.
* Optional: links to presentations, papers, websites, or code repositories related to the talk.

You will also be asked what length of talk you'd like to give: 5, 10, or 20 mintues.  We expect talks to be presented during the morning (PDT time) of the workshop, but the final schedule will depend in part on the number of talk submissions.

You will be notified of your proposal status by email by July 16th.

> [Submit your talk via EasyChair](https://easychair.org/conferences/?conf=pearc20rse).  You will need to make an account first if you do not have one yet.

Reviewer: ```suggestion
[Submit your talk via EasyChair](https://easychair.org/conferences/?conf=pearc20rse).  You will need to make an account first if you do not have one.
```
